db/migrate/20230620095911_add_search_to_keyword.rb

```diff
@@ -0,0 +1,15 @@
+class AddSearchToKeyword < ActiveRecord::Migration[7.0]
+  def change
+    change_table :keywords do |t|
+      t.integer :ads_top_count, null: true
+      t.integer :ads_page_count, null: true
+      t.string :ads_top_urls, array: true, null: true
+      t.integer :result_count, null: true
+      t.string :result_urls, array: true, null: true
+      t.integer :total_link_count, null: true
+      t.string :html, null: true
+      t.references :source, null: true, foreign_key: true
+      t.integer :status, default: 0, null: false
```

Comment:
How about set the default 0 to the `integer` field? I think it's more strict than just leave it as `null`. Basically, if the given record is has `ads_top_count` is `null`, it also can be consider as the `0`

Let separate the `references` column with other column.

```suggestion
      t.integer :ads_top_count, default: 0, null: false
      t.integer :ads_page_count, default: 0, null: false
      t.string :ads_top_urls, array: true, null: false
      t.integer :result_count, default: 0, null: false
      t.string :result_urls, array: true, null: true
      t.integer :total_link_count, default: 0, null: false
      t.string :html, null: true
      t.integer :status, default: 0, null: false
      
      t.references :source, null: true, foreign_key: true
```